### DISCUSSION TITLE
Einlesen von Hosts-Datei für Anzeige in Netzwerk Übersicht

### DISCUSSION THREAD

**Discussion by pascalschmiederer** [Freetz-NG/freetz-ng#330] [Unerfüllte Wünsche] [answered]
Hallo zusammen .

Wäre dies Möglich zum umsetzen? 

Die idee ist: 
Das Einlesen der hosts Datei des Systems um die Hostnamen in der Netzwerkübersicht und in der Mesh Übersicht anzuzeigen. 
Somit könnte man sich die doppelte Namensgebung endlich sparen!

**Comment by freeesty** [comment]
Gibt bereits eine automatische Lösung. Aber eine direkte Integration in Freetz wäre schön.

[](https://www.ip-phone-forum.de/threads/hostname-im-avm-webif-durch-dnsmasq-aufl%C3%B6sen-anzeigen.305896/)

**Reply by pascalschmiederer** [reply]
Wie sieht die automatische Lösung aus? Gibt es hierzu einen Link?

**Comment by freeesty** [comment, marked as answer]
Link aus meinem ersten Post gelöscht?

https://www.ip-phone-forum.de/threads/hostname-im-avm-webif-durch-dnsmasq-aufl%C3%B6sen-anzeigen.305896/

**Comment by fda77** [comment]
Wenn ein Post editiert wurde steht "edited" drüber.
Steht nichts da, hast es wohl einfach vergessen...
Beispiele: https://github.com/Freetz-NG/freetz-ng/discussions/134 und https://github.com/Freetz-NG/freetz-ng/issues/326

Befehle die beim (re)start vom multid ausgeführt werden soll könnt ihr in /tmp/flash/mod/multid.start eintragen

Meiner Erfahrung nach setzt AVM die Hostnamen aber auch während dem Betrieb zurück und ich habs irgendwann aufgegeben...
